ext/AvizExt/viz/clustering.jl

```diff
@@ -17,53 +18,104 @@ function ADRIA.viz.clustered_scenarios(
     data::AbstractMatrix,
     clusters::Vector{Int64};
     fig_opts::Dict=Dict(),
-    axis_opts::Dict=Dict()
-)
+    axis_opts::Dict=Dict(),
+    opts::Dict=Dict()
+)::Figure
     f = Figure(; fig_opts...)
     g = f[1, 1] = GridLayout()
 
-    ADRIA.viz.clustered_scenarios!(g, data, clusters; axis_opts=axis_opts)
+    ADRIA.viz.clustered_scenarios!(g, data, clusters; axis_opts=axis_opts, opts=opts)
 
     return f
 end
 function ADRIA.viz.clustered_scenarios!(
     g::Union{GridLayout,GridPosition},
     data::AbstractMatrix,
     clusters::Vector{Int64};
-    axis_opts::Dict=Dict()
-)
+    axis_opts::Dict=Dict(),
+    opts::Dict=Dict()
+)::Union{GridLayout,GridPosition}
     # Ensure last year is always shown in x-axis
     xtick_vals = get(axis_opts, :xticks, _time_labels(timesteps(data)))
     xtick_rot = get(axis_opts, :xticklabelrotation, 2 / π)
-    ax = Axis(
-        g[1, 1],
-        xticks=xtick_vals,
-        xticklabelrotation=xtick_rot;
-        axis_opts...
-    )
+    ax = Axis(g[1, 1], xticks=xtick_vals, xticklabelrotation=xtick_rot; axis_opts...)
 
-    # Compute cluster colors
-    clusters_colors = _clusters_colors(clusters)
-    unique_cluster_colors = unique(clusters_colors)
+    if get(opts, :method, :series) == :confint
+        _plot_clusters_confint!(g, ax, data, clusters; opts=opts)
+    else
+        _plot_clusters_series!(g, ax, data, clusters; opts=opts)
+    end
+
+    return g
+end
 
+function _plot_clusters_series!(
+    g::Union{GridLayout,GridPosition},
+    ax::Axis,
+    data::AbstractMatrix,
+    clusters::Vector{Int64};
+    opts::Dict=Dict()
+)::Nothing
+    alphas = _get_alphas(clusters)
+    clusters_colors = unique(_get_colors(clusters, alphas))
     leg_entry = Any[]
-    for cluster in unique(clusters)
-        cluster_color = _cluster_color(unique_cluster_colors, clusters, cluster)
+
+    for (idx_c, cluster) in enumerate(unique(clusters))

```

Comment:
Trying to get rid of the `Any[]`:

Something like:

```julia
leg_entry = [
    series!(ax, data[:, clusters.==cluster]', solid_color=clusters_colors[idx_c]) 
        for (idx_c, cluster) in enumerate(unique(clusters))
]
```

Not sure what type the vector would be but good to specify that too